### PR TITLE
[new_profile] DOB Format moved to Config table

### PR DESCRIPTION
**Pull request by ridz1208**
[comment]: # (
Make sure you send your pull request to the right VERSIONNUMBER-dev branch.
It's still possible to change it afterwards and if you are not sure which branch to send to, please consult the CONTRIBUTING.md.
)
[comment]: # (
Please also make sure to indicate in the pull request title the general domain in which your pull request pertains to.
For example, "[Core] Adding CentOS support to Install Script" or "[Document Repository] Adding functionality to group files under Categories".
)
[comment]: # (
Also set appropriate Labels and Milestones as you see fit. Assignee is usually set to whoever may be the right person to test your pull request.
)

- [x] Moved DOB format to the config table to allow project customization

[comment]: # (- [ ] add disclaimer on bottom of page)

- [x] modify test plan

### CODE REVIEW

**Comment by jstirling91** [comment]
Adding caveat as projects that use just Ym will need to alter the path to set the format correct

File: SQL/0000-00-03-ConfigTables.sql [hunk 1]
```diff
@@ -25,7 +25,7 @@ CREATE TABLE `Config` (
 
 -- study
 INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, Label, OrderNumber) VALUES ('study', 'Settings related to details of the study', 1, 0, 'Study', 1);
-INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'additional_user_info', 'Display additional user profile fields on the User accounts page (e.g. Institution, Position, Country, Address)', 1, 0, 'boolean', ID, 'Additional user information', 14 FROM ConfigSettings WHERE Name="study";
+INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'additional_user_info', 'Display additional user profile fields on the User accounts page (e.g. Institution, Position, Country, Address)', 1, 0, 'boolean', ID, 'Additional user information', 15 FROM ConfigSettings WHERE Name="study";
```
Comment (jstirling91) [inline SQL/0000-00-03-ConfigTables.sql]: What are these hardcoded values for?

Comment (ridz1208) [inline SQL/0000-00-03-ConfigTables.sql]: @jstirling91 They are used for ordering but they are pretty annoying. I added the discussion, lets talk about it

**Comment by ridz1208** [comment]
disclaimer ?

**Comment by ridz1208** [comment]
will be discussed on Tuesday

**Comment by ridz1208** [comment]
@driusan please merge

**Comment by ridz1208** [comment]
@driusan please review and merge

**Pull request merged**